src/world.cpp

```diff
@@ -88,23 +85,14 @@ bool World::init(vec2 screen)
 		fprintf(stderr, "Failed to open audio device");
 		return false;
 	}
-	Mix_AllocateChannels(16);

```

Comment:
related to my comment above about the sound effects. I think we might need this in order to play multiple sound effects at the same time